packages/kit/src/core/prerender/crawl.js

```diff
@@ -125,7 +128,9 @@ export function crawl(html) {
 								let escaped = false;
 
 								while (i < html.length) {
-									if (!escaped) {
+									if (escaped) {
+										escaped = false;
```

Comment:
this looks like a bugfix for double escaping. does that need a separate changeset?

Reply:
done